src/Form/QuestionType/QuestionTypeShortAnswer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form\QuestionType;
+
+use Glpi\Application\View\TemplateRenderer;
+use Glpi\Form\Question;
+
+/**
+ * Short answers are single line inputs use to answer simple questions.
+ */
+class QuestionTypeShortAnswer implements QuestionTypeInterface
+{
+    public function renderAdminstrationTemplate(
+        ?Question $question = null,
+        ?string $input_prefix = null
+    ): string {
+        $template = <<<TWIG
+            <input
+                class="form-control mb-2"
+                type="text"
+                name="{{ input_prefix is not null ? input_prefix ~ '[default_value]' : 'default_value' }}"
+                placeholder="{{ __("No default value") }}"
+                value="{{ question is not null ? question.fields.default_value : '' }}"
```

Comment:
It is safer to us the dedicated escape function.
```suggestion
                value="{{ question is not null ? question.fields.default_value|e('html_attr') : '' }}"
```